_includes/hero.html

```diff
@@ -1,59 +1,57 @@
 {% macro render(hero) %}
 
 {% from "post-macros.html" import category_icon as category_icon %}
-    
+
 {% if hero['related_post'] %}
     {% set related_post = get_document('posts', hero['related_post']) %}
 {% endif %}
 
+{% if related_post %}
+    {% set hero_href = ['href=', related_post.permalink] %}
```

Comment:
What do you think about naming this `hero_href_attribute` or `hero_href_attr` to be more specific since it's not just the href value? (Here and below.)